.bitrise/build-and-test.yml

```diff
@@ -0,0 +1,54 @@
+---
+format_version: '11'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: android
+workflows:
+  build-and-test:
+    description: |-
+      # Build the project and run the test (unit and ui).
+    steps:
+      - cache-pull@2: { }
+      - install-missing-android-tools@3:
+          inputs:
+            - gradlew_path: "./gradlew"
+      - gradle-runner@2:
+          title: Checking code documentation
+          inputs:
+            - gradlew_path: "./gradlew"
+            - gradle_task: detekt detektTest
+      - gradle-runner@2:
+          title: Running unit tests
+          inputs:
+            - gradlew_path: "./gradlew"
+            - gradle_task: testDebug
+      - script:
+          title: Checking if the instrumentation tests must run
+          inputs:
+            - content: |-
+                execute_integration_tests=1
+                if [[ $BITRISE_GIT_MESSAGE == *"[integration-tests skip]"* ]]
+                then
+                  execute_integration_tests=0
+                fi
+
+                # execute_integration_tests == 1 -> execute : execute_integration_tests != 1 -> skip
+                envman add --key EXECUTE_INTEGRATION_TESTS --value $execute_integration_tests
+      # - bitrise-run@0.9:
```

Comment:
This step is not needed & we don't have any tests
feel free to delete it & we can add it later on when we have tests